assets/airports/eggw.json

```diff
@@ -1,1266 +1,1638 @@
+        "DOMUT": ["N50d15m43", "W001d40m09"],
+        "DTY"  : ["N52d10m48.51", "W001d06m49.64"],
+        "DVR"  : ["N51d09m45.44", "E001d21m32.72"],
+        "ELVOS": ["N52d42m01", "W001d18m25"],
+        "EMKAD": ["N51d14m50", "E000d42m34"],
+        "EMKUK": ["N51d58m37", "W001d21m15"],
+        "ETVAX": ["N50d58m07", "E000d35m56"],
+        "EVTES": ["N51d04m55", "W001d12m57"],
+        "EXARO": ["N51d00m03", "W001d22m50"],
+        "FINCH": ["N51d33m36", "W000d10m59"],
+        "FINMA": ["N51d59m47", "W001d03m13"],
+        "GIBSO": ["N50d44m59", "W002d31m07"],
+        "GILDA": ["N51d36m06", "E000d35m01"],
+        "GOGSI": ["N51d17m27", "W001d00m02"],
+        "GOXUL": ["N51d21m32", "W000d10m33"],
+        "GWE16": ["N51d46m22.04", "W000d15m46.78"],
+        "GWE19": ["N51d45m40.56", "W000d11m04.88"],
+        "GWE37": ["N51d42m59.91", "E000d16m58.25"],
+        "GWS01": ["N51d51m19.98", "W000d25m14.51"],
+        "GWS06": ["N51d47m05.83", "W000d29m28.41"],
+        "GWS12": ["N51d46m56.85", "W000d19m44.38"],
+        "HAZEL": ["N51d00m19", "W000d59m04"],
+        "HEMEL": ["N51d48m20", "W000d25m10"],
+        "HEN"  : ["N51d45m35.07", "W000d47m25.05"],
+        "HON"  : ["N52d21m24.04", "W001d39m49.41"],
+        "IDESI": ["N51d53m52", "E001d53m08"],
+        "JACKO": ["N51d44m09", "E001d25m36"],
+        "KAPEX": ["N50d40m30", "W001d51m00"],
+        "KATHY": ["N50d31m14", "W001d20m00"],
+        "KENET": ["N51d31m14", "W001d27m18"],
+        "KONAN": ["N51d07m51", "E002d00m00"],
+        "LAPRA": ["N52d07m07", "E001d12m36"],
+        "LELNA": ["N49d58m57", "W002d06m23"],
+        "LISTO": ["N53d08m36", "W002d11m57"],
+        "LIVSU": ["N51d41m49", "E000d50m58"],
+        "LOGAN": ["N51d44m51", "E001d36m43"],
+        "LOREL": ["N52d00m50", "W000d03m10"],
+        "LAM"  : ["N51d38m45.69", "E000d09m06.13"],
+        "LUT"  : ["N51d53m40.68", "W000d15m09.18"],
+        "MANGO": ["N51d41m34", "E000d47m26"],
+        "MARUK": ["N50d14m47", "W002d04m20"],
+        "MATCH": ["N51d46m45", "E000d15m00"],
+        "MAY"  : ["N51d01m01.86", "E000d06m58.04"],
+        "MCT"  : ["N53d21m25.29", "W002d15m44.24"],
+        "MEGEL": ["N52d20m58", "E002d04m25"],
+        "MID"  : ["N51d03m14.23", "W000d37m30.01"],
+        "MIMBI": ["N51d32m57", "W001d41m36"],
+        "NEDUL": ["N50d39m58", "W001d32m52"],
+        "NEPNA": ["N51d29m58.33", "E000d26m56.55"],
+        "NIGIT": ["N51d18m47", "W001d10m15"],
+        "NORRY": ["N51d28m47", "W001d07m24"],
+        "NUBRI": ["N51d20m17", "W001d13m56"],
+        "OCK"  : ["N51d18m18.17", "W000d26m49.86"],
+        "OLNEY": ["N52d07m40", "W000d44m03"],
+        "ORTAC": ["N49d59m57", "W002d00m18"],
+        "PEDIG": ["N52d44m48", "W001d43m10"],
+        "PEPIS": ["N51d11m48", "W001d14m37"],
+        "REDFA": ["N52d06m53", "E002d29m17"],
+        "RODNI": ["N51d42m57", "W000d51m42"],
+        "ROGBI": ["N52d17m47", "W001d20m06"],
+        "ROTNO": ["N50d51m26", "E000d46m41"],
+        "SA
```

Comment:
(line 1409)
Something isn't quite right about the DET6C departure (also applies to MATCH1C)
![det6c](https://user-images.githubusercontent.com/13125311/36357834-a0d6e58a-14fb-11e8-99b7-551ca31f282e.PNG)